content/vault/v1.18.x/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -18,6 +18,48 @@ Vault 1.15. **Please read carefully**.
 
 ## Important changes
 
+### JSON Payload Limits
+
+| Change       | Affected version | Vault edition
+| ------------ | ---------------- | -------------
+| New behavior | 1.16.25           | All
```

Comment:
Could you add all the versions?

Reply:
Yes, ideally we'd have the full list of affected versions everywhere this appears in the docs.

Reply:
We need to start being more precise about the affected versions. If the new behavior is introduced in 1.16.25, then we should say 1.16.25+ not just "1.16.x". Same for breaking changes (when we introduced the change) and known issues (the earliest versions of that release known to include the issue).